apps/web/src/pages/[document].tsx

```diff
@@ -48,6 +64,7 @@ const DocumentPage: NextPage<Props> = ({ page: initialPage }) => {
   const [page, setPage] = useState<ReturnType<typeof deserializePage> | null>(
     null
   );
+  const [metaTags, setMetaTags] = useState<MetaTagsProps | null>(null);
```

Comment:
Instead of storing this in state, could we derive it from the `page` state on render?

Reply:
```tsx
<meta property="og:title" content={metaTags.title} />
```
becomes
 ```tsx
<meta property="og:title" content={page.title} />
```

Reply:
Hm, yeah, we can, but I think it's also nicer to format it with the state. It's friendlier for the eyes to read `metaTags.description` than `page?.content[0]?.text`.